lib/between_meals/repo/git.rb

```diff
@@ -58,7 +58,7 @@ def last_author
       end
 
       def head_parents
-        @repo.head.target.parents
+        @repo.head.target.parents.map {|x| {:time => x.time}}
```

Comment:
we're not the only people writing plugins. It's one thing to change the API, another to drop something the API provides.